### ISSUE TITLE
Permission denied: './data/GameSaveDir.txt'

### ISSUE THREAD

**Issue by Ariescyn** [Ariescyn/EldenRing-Save-Manager#11]
User reported the following Traceback after successful update from v1.145 to v1.46.
Waiting to hear back from the user on which OS this occurred. 

Traceback (most recent call last): File "SaveManager.py", line 21, in <module>
PermissionError: [Errno 13] Permission denied: './data/GameSaveDir.txt'

Each release does not contain GameSaveDir.txt so that when the app updates, it doesn't overwrite the users existing settings. So the executable and text file after update must no longer belong to the same user/group?

**Comment by VinegarLove** [comment]
It's probably as you say.
There are a lot of edge cases that can lead to this... We have to ask some questions.

1. Did the user updated the manager using the "update" script or from the nexus? Maybe the updater, as you said, has problem with users/group.
2. Which version of windows? Home edition and stuff have locked permission, so if it gets executed in certain folder it can lead to issues. "Documents" folder should be a safe folder, but i don't know much about it.
3. ./data/GameSaveDir.txt was written with a different permission.

If i can reproduce the issue i'll let you know.

**Comment by VinegarLove** [comment]
Quoting:

There are three main reasons behind the permission denied error. 

1. Insufficient privileges on the file or for Python
2. Passing a folder instead of file
3. File is already open by other process

**Comment by Ariescyn** [comment]
So one user said that putting the save manager on the same hard drive as the game fixed it for him

**Comment by VinegarLove** [comment]
At this point i would put a Popup in the application if it fails (with try / except is possible) and suggest what the user did.
It should be something like:

`try:
      with open(gamesavedir_txt, 'w') as fh:
           fh.write(eldenring_savedata_dir)
except IOError as e:
      <inform the user>
`

**Comment by Ariescyn** [comment]
Here's a response from one of the users on Nexus:

Elden Ring is saved on a Drive called (B:) which almost exclusively contains gamefiles while I was initially running the save manager from my desktop, which would be my primary (C:) drive, when I got the error.
But I just tried to replicate the conditions of the error and once again tried to run the manager from my desktop, I got the same error message, though unlike last time now my Antivirus AVG rang the alarm about the save manager trying to access a certain Elden Ring file, I could now tell it to allow the process and can now run the save manager without any trouble or any error messages form my primary drive as well. So maybe the issue is actually just a specific Antivirus?

Hope it helps.
